frontend/apps/reader/readerui.lua

```diff
@@ -329,6 +330,7 @@ end
 
 function ReaderUI:showReader(file)
     DEBUG("show reader ui")
+    ReadHistory:addItem(file)
```

Comment:
I doubt it matters much on the machine end; I'd be worried more about whether the GUI allows for dealing efficiently with more than a page or three and whether it adds anything for the user. If I know it's a book I read sometime last year I use my well-organized folder structure or (rarely, if ever, even though I wrote it) the search functionality. That raises the question of whether search in history would be worthwhile, but again I'm inclined to think that'd add extra GUI complexity without really offering anything extra. To summarize, I think even as "little" as two or three hundred should suffice.

tl;dr I'm fine with 500, think 250 would suffice, and think 1000 wouldn't really profit anyone.